rollup.config.js

```diff
@@ -82,7 +82,6 @@ export default [
         jsnext: true
       }),
       babel({
-        exclude: 'node_modules/**',
```

Comment:
Yeah, I tried explicitly including the modules needed, but it didn't work. I haven't looked into it deeply, but I suspect `exclude` trumps `include` for some reason.

```
exclude: 'node_modules/**',
include: ['node_modules/swiper/**', 'node_modules/dom7/**']
```

and variations, didn't work

Reply:
OK. I'll QA a bit to see if including all node modules doesn't have an impact on anything...

Reply:
Not sure if `include`/`exclude` are meant to be used together, but yes - if something is excluded it will not be included.

What worked for me was just using include: `include: ['static/**', 'node_modules/swiper/**', 'node_modules/dom7/**'],` to include our scripts and swiper + dom7.

On the other hand I'm not sure if there are any downsides of babeling everything from node_modules - other than that we don't need to do it, we are transpiling code that doesn't need it and adding to build time.

Reply:
Yeah, the extra build time wasn't noticeable so I figured it's a small compromise.

Does including as you did exclude all other node_modules? If the file size is smaller/ build time is shorter it's probably worth it.

Reply:
Other node modues were excluded from babel (while of course still in the bundle if needed). If I recall correctly using babel on everything actually resulted in smaller file (by a small amount).

So, while I feel a bit reluctant to babel all the things, I don't really have any more arguments. ;)
It seems to be working fine.

Reply:
:D